init.sql

```diff
@@ -0,0 +1,26 @@
+CREATE ROLE librarian WITH LOGIN SUPERUSER PASSWORD 'librarian';
+ALTER DATABASE library OWNER TO librarian;
+
+CREATE TABLE person (
+    id          INT         PRIMARY KEY,
+    firstName   VARCHAR(32),
+    lastName    VARCHAR(32),
+    birthdate   DATE,
+    hometown    VARCHAR(32)
+);
+
+CREATE TYPE phone_type AS ENUM ('work', 'home', 'personal');
+CREATE TABLE phone (
+    number      INT         PRIMARY KEY,
```

Comment:
Don't use phone number as primary key, because, for example, work phone can be one for many.